cirq/ops/controlled_gate_test.py

```diff
@@ -18,7 +18,7 @@
 
 import cirq
 from cirq.type_workarounds import NotImplementedType
-
+from cirq import protocols
```

Comment:
for tests you should try to just use cirq.CircuitDiagramInfoArgs (so that it is like what users write).  So delete this import should make you fix the use of protocols.X below.

Reply:
sounds good, thank you!